src/components/NavHeader.js

```diff
@@ -0,0 +1,42 @@
+import { Link } from 'react-router-dom';
+import '../styles/NavHeader.css';
+import React from 'react';
+// import { NavLink } from 'react-router-dom';
+// import planetImg from '../public/planet2.png';
```

Comment:
- [ ] it would be better to remove the commented code in order to keep your file clean and pro